src/set_theory/zfc.lean

```diff
@@ -215,6 +216,8 @@ def of_nat : ℕ → pSet
 /-- The von Neumann ordinal ω -/
 def omega : pSet := ⟨ulift ℕ, λ n, of_nat n.down⟩
 
+-- Todo (Vi): add general Neumann ordinals.
+
```

Comment:
These TODO comments seem a bit weird to me; either remove `Vi` or use your full github username instead